index.html

```diff
@@ -32,5 +32,159 @@ <h3>LET'S CONNECT</h3>
                 </ul>
             </section>
         </div>
+        <section class="work">
+            <div class="projects">
+                <img class="snapshots" id="snap1" src="snap.svg">
+                <div class="info_block">
+                    <div class="projo">
+                        <h4 class="project_name">Tonic</h4>
+                        <ul class="deatils">
+                            <li class="detail">CANOPY</li>
+                            <li class="detail"><img src="Counter.svg"></li>
+                            <li class="detail">Back End Dev</li>
+                            <li class="detail"><img src="Counter.svg"></li>
+                            <li class="detail">2023</li>
+                        </ul>
+                    </div>
+                    <p class="info">A daily selection of privately personalized reads; no accounts
+                        or sign-ups required.
+                    </p>
+                    <ul class="languages">
+                        <li class="language" id="html">HTML</li>
+                        <li class="language" id="css">CSS</li>
+                        <li class="language" id="javascript">Javascript</li>
+                    </ul>
+                    <button class="button" type="button">see project</button>
+                </div>
+            </div>
+            <div class="projects">
+                <img class="snapshots" id="snap2" src="snap1.svg">
+                <div class="info_block">
+                    <div class="projo">
+                        <h4 class="project_name">Multi-Post Stories</h4>
+                        <ul class="deatils">
+                            <li class="detail">CANOPY</li>
+                            <li class="detail"><img src="Counter.svg"></li>
+                            <li class="detail">Back End Dev</li>
+                            <li class="detail"><img src="Counter.svg"></li>
+                            <li class="detail">2023</li>
+                        </ul>
+                    </div>
+                    <p class="info">A daily selection of privately personalized reads; no accounts
+                        or sign-ups required.
+                    </p>
```

Comment:
- [ ] I would suggest you to align the languages to be on the same line with the tonic project card or modal,this will make your website look professional like the figma template kindly fix this .
- [ ] see screenshot
![ddd](https://github.com/Omwomo/Microverse-web-portfolio-project/assets/92458236/20fe580b-f6d7-445e-a92f-609d2963f5d2)